src/olympia/devhub/templates/devhub/addons/submit/done.html

```diff
@@ -32,10 +38,11 @@
         _("Edit version {0}")|fe(uploaded_version.version) }}</a>
   </p>
 {% else %}
-  <h3>{{ _("Version Signed") }}</h3>
+
+  <h3>{{ _("%s Signed"|format(submit_page)|title) }}</h3>
```

Comment:
D'oh! I forgot about l10n when I was being clever and re-using the `submit_page` variable.  It's going to be clearer with an `{% if... `.